presentation/src/main/res/values-bn/strings.xml

```diff
@@ -366,6 +366,9 @@
     </plurals>
     <string name="notification_message_failed_title">বার্তা পাঠাতে ব্যর্থ</string>
     <string name="notification_message_failed_text">%s এর কাছে বার্তাটি পাঠাতে ব্যর্থ</string>
+    <string name="open_source_credit_title">Open Source Credit</string>
+    <string name="up_open_source_link_title">Unplugged Source Code</string>
```

Comment:
Isn't it easier to use default values/strings.xml when no translation is available?

Reply:
When I added these strings it asked me to which locals to add them so I chose all of them, so it will present these in all languages and will not have issues with missing strings

Reply:
No need to add to all languages, leave only the default one (English)